packages/styling/src/styles/IconCodes.ts

```diff
@@ -1,2830 +1,1114 @@
 export const IconCodes = {
```

Comment:
To review the changes, see the third commit rather than this summary of changes. It's less noisy.